enterprise/backend/test/metabase_enterprise/serialization/v2/e2e_test.clj

```diff
@@ -735,3 +738,25 @@
                     (is (thrown-with-msg? Exception #"Please upgrade"
                                           (cmd/v2-load! dump-dir {}))
                         "throws")))))))))))
+
+(deftest every-model-is-supported-test
```

Comment:
I think this belongs in `models_test` - there's nothing "e2e" about this test.

Reply:
🤦 right